examples/benchmarks/bert/src/bert_layers.py

```diff
@@ -54,11 +54,26 @@
                                            SequenceClassifierOutput)
 from transformers.models.bert.modeling_bert import BertPreTrainedModel
 
+IMPL_USE_FLASH2 = False
```

Comment:
Can we pass this as a flag via the yamls?

Reply:
I mean we could, but at what point we are just replicating llm-foundary. The idea is people are more likely to have the latest Flash Attention installed then a highly specific version of triton setup.

Reply:
tbh i think you can just require the new version of fa, and get rid of the triton version in here. agree that we should not recreate foundry here.